DI/Package.swift

```diff
@@ -0,0 +1,31 @@
+// swift-tools-version:5.3
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    name: "DI",
+    platforms: [.iOS(.v12)],
+    products: [
+        // Products define the executables and libraries a package produces, and make them visible to other packages.
+        .library(
+            name: "DI",
+            targets: ["DI"]),
+    ],
+    dependencies: [
+        // Dependencies declare other packages that this package depends on.
+        // .package(url: /* package url */, from: "1.0.0"),
+        //.package(name: "Data", path: "../Data"),
+        //.package(name: "Domain", path: "../Domain")
+    ],
+    targets: [
+        // Targets are the basic building blocks of a package. A target can define a module or a test suite.
+        // Targets can depend on other targets in this package, and on products in packages this package depends on.
+        .target(
+            name: "DI",
+            dependencies: [/*"Data", "Domain"*/]),
+        .testTarget(
+            name: "DITests",
+            dependencies: ["DI"]),
+    ]
+)
```

Comment:
What is that DI package?

Reply:
It was because I wanted to make a dependency injector but it didn't work, I deleted it there